app/src/main/java/com/asfoundation/wallet/ui/widget/holder/TransactionDetailsHolder.java

```diff
@@ -18,8 +23,9 @@
  * transaction.
  */
 public class TransactionDetailsHolder extends BinderViewHolder<Operation>
-    implements View.OnClickListener {
+    implements OnClickListener {
 
+  private static final int DEFAULT_SCALE = 8;
   public static final int VIEW_TYPE = 1007;
```

Comment:
not being used, so it can be removed

Reply:
Sorry didn't see it. It was already there. Going to remove it.